way_to_home/tests/unittests/custom_user/testviews.py

```diff
@@ -207,11 +207,8 @@ def test_log_in_validator_fail(self):
     def test_google_auth_success(self):
         """Provides test for a (GET) request to authenticate via Google."""
         url = reverse('auth_google')
-        expected_reverse = reverse('sign_in_google')
-        with mock.patch('requests_oauthlib.OAuth2Session.authorization_url') as authorization_url:
-            authorization_url.return_value = [expected_reverse]
-            response = self.client.get(url, follow=True)
-            self.assertEquals(response.redirect_chain[0], (expected_reverse, 302))
+        response = self.client.get(url)
+        self.assertEquals(response.status_code, 200)
```

Comment:
In my opinion, you should stiluse mock here to test json that your view returns.

Reply:
this should work:
        ```
expected_data={'url': 'somwhere.com/'{
    with mock.patch('requests_oauthlib.OAuth2Session.authorization_url') as authorization_url:
            authorization_url.return_value = 'somwhere.com/'
            response = self.client.get(url)
            self.assertJSONEqual(json.dumps(expected_data), json.loads(response.content))
```

Reply:
done